tests/unit/test_find_urls.py

```diff
@@ -15,6 +15,12 @@
 
     ("Let's have text without URLs.",
      []),
+    
+    ("Comma in URL 1,420.00.zip",
+     []),
```

Comment:
Same here as I commented with email.
I would prefer to get `[420.00.zip]` as URL rather than empty list